config/gitleaks.toml

```diff
@@ -482,7 +482,7 @@ keywords = [
 [[rules]]
 description = "Generic API Key"
 id = "generic-api-key"
-regex = '''(?i)(?:key|api|token|secret|client|passwd|password|auth|access)(?:[0-9a-z\-_\t .]{0,20})(?:[\s|']|[\s|"]){0,3}(?:=|>|:=|\|\|:|<=|=>|:)(?:'|\"|\s|=|\x60){0,5}([0-9a-z\-_.=]{10,150})(?:['|\"|\n|\r|\s|\x60|;]|$)'''
+regex = '''(?i)(?:key|api|token|secret|client|passwd|password|auth|access)(?:[0-9a-z\-_\t .]{0,20})(?:[\s|']|[\s|"]){0,3}(?:=|>|:=|\|\|:|<=|=>|:)(?:'|\"|\s|=|\x60){0,5}([0-9a-z\-_.=!#$%&()*+,/:;<>?@[\\\]^{|}~]{10,150})(?:['|\"|\n|\r|\s|\x60|;]|$)'''
```

Comment:
You need to escape the slash like:
```(?i)(?:key|api|token|secret|client|passwd|password|auth|access)(?:[0-9a-z\-_\t .]{0,20})(?:[\s|']|[\s|"]){0,3}(?:=|>|:=|\|\|:|<=|=>|:)(?:'|\"|\s|=|\x60){0,5}([0-9a-z\-_.=!#$%&()*+,\/:;<>?@[\\\]^{|}~]{10,150})(?:['|\"|\n|\r|\s|\x60|;]|$)```

**Original:**
(?i)(?:key|api|token|secret|client|passwd|password|auth|access)(?:[0-9a-z\-_\t .]{0,20})(?:[\s|']|[\s|"]){0,3}(?:=|>|:=|\|\|:|<=|=>|:)(?:'|\"|\s|=|\x60){0,5}([0-9a-z\-_.=!#$%&()*+, **===>  /   <===**:;<>?@[\\\]^{|}~]{10,150})(?:['|\"|\n|\r|\s|\x60|;]|$)

Reply:
At least according to regex101, the syntax should be fine, also to the unit tests went through. However, I changed it anyway, to be more on the save side.